packages/calling/src/CallingClient/line/index.ts

```diff
@@ -131,7 +127,9 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
    */
   public async register() {
     await this.#mutex.runExclusive(async () => {
-      this.registration.setStatus(MobiusStatus.DEFAULT);
+      if (this.registration.getStatus() !== RegistrationStatus.IDLE) {
```

Comment:
let's remove this code and check what happens

Reply:
this is removed